bitnami/kiam/values.yaml

```
@@ -489,10 +489,10 @@ agent:
   ##
   podSecurityPolicy:
     create: true
-    allowedHostPaths:
+    allowedHostPaths: {}
```

```suggestion
    allowedHostPaths: []
```
if this is not the valid type, please check https://github.com/bitnami/charts/blob/907669f327d7b320119ae87b1fad539d26860eca/bitnami/kiam/values.yaml#L101 to be consistent